.claude/skills/basilica-localnet-debug/SKILL.md

```
@@ -0,0 +1,284 @@
+name: basilica-localnet-debug
+description: Debug Basilica localnet services using existing scripts and Docker Compose commands
+---
+
+# Basilica Localnet Debugging
+
+## Architecture Reference
+
+### Services
+
+| Service | Container Name | IP Address | Ports |
+|---------|---------------|------------|-------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)